gradio/components.py

```diff
@@ -3257,7 +3257,8 @@ def postprocess(
                 index = 0
                 entities = sorted(entities, key=lambda x: x["start"])
                 for entity in entities:
-                    list_format.append((text[index : entity["start"]], None))
+                    if index != entity["start"]:
```

Comment:
Just to be clear - users would still need to set `combine_adjacent=True` right?

Reply:
Yes, that's correct